src/io/utilities/parseSBMLNotesField2012.m

```diff
@@ -63,15 +63,15 @@
     elseif (regexp(fieldStr,'^FORMULA'))
         formula = regexprep(strrep(fieldStr,'FORMULA:',''),'^(\s)+','');
     elseif (regexp(fieldStr,'^CHARGE'))
-        charge = str2num(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
+        charge = str2double(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
     elseif (regexp(fieldStr,'AUTHORS'))
         if isempty(citation)
             citation = strcat(regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         else
             citation = strcat(citation,';',regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         end
     elseif (regexp(fieldStr,'^Confidence Level'))
-        confidenceScore = regexprep(strrep(fieldStr,'Confidence Level:',''),'^(\s)+','');
+        [matches, tmpTokens] = regexpi(fieldStr, 'Confidence[ _]Level: (\w+)', 'match', 'tokens');
```

Comment:
There's a few lines missing here, so matches and tokens aren't used.
    if (~isempty(matches))
        confidenceScore = str2double(tmpTokens{1}{1});
    end